content/en/security/cloud_security_management/_index.md

```diff
@@ -54,6 +53,12 @@ Cloud Security Management includes [Cloud Security Posture Management (CSPM)](#c
 
 {{< img src="security/cws_overview.png" alt="Cloud Workload Security coverage views in Datadog" width="100%">}}
 
+## CSM Identity Risks
+
+[CSM Identity Risks][4] provides in-depth visibility into your organization's AWS IAM risks. Enabling you to detect and resolve identity risks on an ongoing basis, CSM Identity Risks helps secure your cloud infrastructure from IAM-based attacks.
```

Comment:
[Cloud Infrastructure Entitlement Management (CIEM)][4], surfaced in CSM (hyperlink) as "Identity Risks", provides in-depth visibility into your organization's AWS IAM risks. It enables you to detect and resolve identity risks on an ongoing basis to secure your cloud infrastructure from IAM-based attacks.